.github/workflows/codeql.yml

```diff
@@ -0,0 +1,57 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "7.0", "3.13", "3.14", "3.15", "3.16" ]
```

Comment:
7.0 and 3.15 only I would think.

Reply:
Actually we might want to exclude 3.15 even — any dis/agreement on that? There's no `lgtm.yml` in our 3.15 branch.